components/common/ConnectWallet/WalletAssetMenu.vue

```diff
@@ -13,21 +13,31 @@
       </a>
     </div>
     <div
-      class="wallet-asset-footer is-flex is-justify-content-space-between py-5 is-size-7 has-text-grey">
+      class="wallet-asset-footer is-flex py-5 is-size-7 has-text-grey"
+      :class="
+        isMobileDevice
+          ? 'is-justify-content-center'
+          : 'is-justify-content-space-between'
+      ">
```

Comment:
css better than js for this
use `@include mobile` to toggle justify-content
https://bulma.io/documentation/utilities/responsive-mixins/#named-mixins